src/interface/meta.thrift

```diff
@@ -184,9 +183,10 @@ struct CreateTagReq {
 }
 
 struct AlterTagReq {
-    1: common.GraphSpaceID    space_id,
-    2: string                 tag_name,
-    3: list<AlterSchemaItem>  tag_items,
+    1: common.GraphSpaceID      space_id,
+    2: string                   tag_name,
+    3: list<AlterSchemaOption>  schema_options,
```

Comment:
Ditto

Reply:
I think this way is clearer.
We only modify the column in schem here, but cannot modify the propertyin schema.

Reply:
Ditto.   
For any PRs,   we have a simple principle:  Do do any meaningless changes.  If you think you have strong reasons for it,  please open another issue,  let's discuss it there.

Reply:
I will correct it according to your opinion here.